README.md

```diff
@@ -1,160 +1,90 @@
-NodeJS & NPM
-------------
-To install via npm:
+jStat can be used in the browser. The `jStat` object will be added to the window. For example:
 
 ```
-npm install --save jStat
-```
-
-When loading under Node be sure to reference the child object.
+<script src="https://cdn.jsdelivr.net/npm/@handsontable/jstat/dist/jstat.min.js"></script> <!-- include jStat, from the CDN or otherwise -->
+<script>
+...
+var jstat = this.jStat(dataset); // jStat will be added to the window
+...
+data[i]['cum'] = jstat.normal(jstat.mean(), jstat.stdev()).cdf(data[i].x);
+...
+</script>
 
-```
-var jStat = require('jStat').jStat;
 ```
 
-RequireJS Shim
---------------
+Using jStat in Node environment.
 
-For RequireJS not only `exports` but also `init` function must be specified.
 ```
-requirejs.config({
-  paths: {
-    'jstat': 'path/to/jstat/dist/jstat.min'
-  },
-  shim: {
-    jstat: {
-      exports: ['j$', 'jStat'],
-      init: function () {
-        return {
-          j$: j$,
-          jStat: jStat
-        };
-      }
-    }
-  }
-});
+const jStat = require('@handsontable/jstat');
+...
+data[i]['cum'] = jStat.normal(jStat.mean(), jStat.stdev()).cdf(data[i].x);
+...
 ```
 
-Build Prerequisites
--------------------
-
-In order to build jStat, you need to have GNU make 3.8 or later, Node.js 0.2 or later, and git 1.7 or later.
-(Earlier versions might work OK, but are not tested.)
-
-Windows users have two options:
+## Building jStat
 
-1. Install [msysgit](https://code.google.com/p/msysgit/) (Full installer for official Git),
-   [GNU make for Windows](http://gnuwin32.sourceforge.net/packages/make.htm), and a
-   [binary version of Node.js](http://node-js.prcn.co.cc/). Make sure all three packages are installed to the same
-   location (by default, this is C:\Program Files\Git).
-2. Install [Cygwin](http://cygwin.com/) (make sure you install the git, make, and which packages), then either follow
-   the [Node.js build instructions](https://github.com/ry/node/wiki/Building-node.js-on-Cygwin-%28Windows%29) or install
-   the [binary version of Node.js](http://node-js.prcn.co.cc/).
-
-Mac OS users should install Xcode (comes on your Mac OS install DVD, or downloadable from
-[Apple's Xcode site](http://developer.apple.com/technologies/xcode.html)) and
-[http://mxcl.github.com/homebrew/](Homebrew). Once Homebrew is installed, run `brew install git` to install git,
-and `brew install node` to install Node.js.
-
-Linux/BSD users should use their appropriate package managers to install make, git, and node, or build from source
-if you swing that way.
-
-
-Building jStat
---------------
-
-First, clone a copy of the jStat git repo by running `git clone git://github.com/jstat/jstat.git`.
+First, clone a copy of the jStat git repo by running `git clone https://github.com/handsontable/jstat.git`.
 
 To download all necessary libraries run `npm install`.
 
 Then, to get a complete, minified version of jStat and all documentation, simply `cd` to the `jstat` directory and
```

Comment:
We can fix this too, it says to run `npm install` and then "`cd` to the `jstat` directory" to call more commands. `cd` should be first, then `npm install` and then `npm run all`.